webrtc.html

```diff
@@ -2783,6 +2783,29 @@
           <dd>
             <p>An array containing parameters for RTP encodings of media.</p>
           </dd>
+
+          <dt>sequence&lt;RTCRtpCodecParameters&gt; codecs</dt>
+
+          <dd>
+            <p>An array containing codecs that may be used to send RTP
+               encodings.  These can be read
+               from <code>RTCRtpSender.getParameters</code>, and can
+               be changed by <code>RTCRtpSender.setParameters</code>,
+               but the only changes allowed are to either reorder the
+               codecs or remove codecs.  Codecs may not be added or
+               changed.
```

Comment:
We should really text to the description of the setParameters() function instead of having such text here. For example what is allowed to change, how possible errors are reported and if the RTCPeerConnection needs to be marked as needing negotiation.